src/data/README.md

```diff
@@ -1,7 +1,23 @@
-# Data
+# OpenThaiGPT Dataset Pipeline Overview
 
 Folder `openthaigpt_pretraining_data` contains the preprocess function, EDA notebook, and examples for each dataset
 
 Folder `notebook` contains the EDA or other experiment notebook.
 
 Folder `scripts` contains the main program which will call function in folder `openthaigpt_pretraining_data`
+
+## Description
+
+The OpenThaiGPT Overview dataset pipeline is meticulously designed to fully preprocess raw data, extract essential information, and prepare the dataset in a trainable format. Here's an extended breakdown of its key components:
+
+## Workflow
```

Comment:
Can we have some diagram to explain the worlflow. We may export it from our figjam